### PR TITLE
Added organization on FossGraphView

### PR DESCRIPTION
**Pull request by Cheshiriks**
- added organization on FossGraphView
org

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt [hunk 1]
```diff
@@ -207,6 +210,28 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
                                 +"${vulnerability.relatedLink}"
                             }
                         }
+                        if (vulnerability.organization.isNotNull()) {
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt:203]: suggestion to rewrite to `?.run`
```suggestion
                        vulnerability.organization?.run { organization ->
                            hr { }
                            h6 {
                                className = ClassName("font-weight-bold text-primary mb-4")
                                +"Organization"
                            }
                            Link {
                                img {
                                    className =
                                            ClassName("avatar avatar-user width-full border color-bg-default rounded-circle")
                                    src = organization.avatar?.let {
                                        "/api/$v1/avatar$it"
                                    } ?: "img/company.svg"
                                    style = jso {
                                        height = 2.rem
                                        width = 2.rem
                                    }
                                }
                                to = "/${organization.name}"
                                +" ${organization.name}"
                            }
                        }
```

Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt]: Fixed

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt [hunk 2]
```diff
@@ -197,6 +199,28 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
...
+                            hr { }
+                            h6 {
+                                className = ClassName("font-weight-bold text-primary mb-4")
+                                +"Organization"
+                            }
+                            Link {
+                                img {
+                                    className =
+                                            ClassName("avatar avatar-user width-full border color-bg-default rounded-circle")
+                                    src = vulnerability.organization?.avatar?.let {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt:212]:
```suggestion
                                    src = avatar?.let {
```

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt]: Fixed

**Pull request merged**